src/main/java/org/sonarsource/auth/github/GitHubIdentityProvider.java

```diff
@@ -128,6 +134,44 @@ private GsonUser getUser(OAuthService scribe, Token accessToken) {
     return GsonTeams.parse(responseBody);
   }
 
+  public boolean isOrganizationMembershipRequired() {
+    return !settings.organizations().isEmpty();
+  }
+
+  private boolean isOrganizationsMember(Token accessToken, String login) {
+    for (String organization : settings.organizations()) {
+      if (isOrganizationMember(accessToken, organization, login)) {
+        return true;
+      }
+    }
+    return false;
+  }
+
+  /**
+   * Check to see that login is a member of organization.
+   * https://developer.github.com/v3/orgs/members/#response-if-requester-is-an-organization-member-and-user-is-a-member
+   */
+  private boolean isOrganizationMember(Token accessToken, String organization, String login) {
+    String requestUrl = settings.apiURL() + format("orgs/%s/members/%s", organization, login);
+    OAuthService scribe = new ServiceBuilder()
+            .provider(scribeApi)
+            .apiKey(settings.clientId())
+            .apiSecret(settings.clientSecret())
+            .build();
+    OAuthRequest request = new OAuthRequest(Verb.GET, requestUrl, scribe);
```

Comment:
I would have used it but I don't want the response body.  I need the response code.  

The `isSuccessful` (https://github.com/scribejava/scribejava/blob/scribejava-2.1.0/scribejava-core/src/main/java/com/github/scribejava/core/model/Response.java#L53-L55) test in that method also yields an incorrect ISE in the scenarios when GitHub responds with a `302` or `404` to indicate that a login is _not_ a member of an organization.

I didn't want to conflate the current `executeRequest` logic with this so I contained it within the `isOrganizationMember` method.